build-plugin/README.md

```diff
@@ -88,11 +89,34 @@ jobs:
           attestation: true # new line
 ```
 
+## Changelog Generation
+
+If you pass `use_changelog_generator: true` to the action, it will:
+
+1. Generate a changelog using the github-changelog-generator tool
+1. Commit the updated changelog back to your repository's default branch
+
+This feature helps maintain a well-documented history of changes for your plugin. The changelog will include information about merged pull requests, closed issues, and any labeled enhancements or features.
+
+### Add changelog generation to your workflow
+
+To enable changelog generation in your workflow:
+
+```yaml
+- uses: grafana/plugin-actions/build-plugin@main
+  with:
+    policy_token: ${{ secrets.GRAFANA_ACCESS_POLICY_TOKEN }}
+    use_changelog_generator: true
+```
+
+Note: This will commit changes to your repository's default branch, so ensure your workflow has the necessary permissions.
```

Comment:
This might fall short on details. We should be more clear what permissions they need in their token.